pkg/acsengine/defaults.go

```diff
@@ -26,7 +26,7 @@ var (
 		DCOS173BootstrapDownloadURL:     fmt.Sprintf(MsecndDCOSBootstrapDownloadURL, "testing", "df308b6fc3bd91e1277baa5a3db928ae70964722"),
 		DCOS188BootstrapDownloadURL:     fmt.Sprintf(AzureEdgeDCOSBootstrapDownloadURL, "stable", "5df43052907c021eeb5de145419a3da1898c58a5"),
 		DCOS190BootstrapDownloadURL:     fmt.Sprintf(AzureEdgeDCOSBootstrapDownloadURL, "stable", "58fd0833ce81b6244fc73bf65b5deb43217b0bd7"),
-		DCOSWindowsBootstrapDownloadURL: "https://dcosdevstorage.blob.core.windows.net/dcos-windows",
+		DCOSWindowsBootstrapDownloadURL: "http://dcos-win.westus.cloudapp.azure.com/dcos-windows/stable/",
```

Comment:
@yakman2020 to speak to the "to use vlabs or not vlabs" question, let me add a slightly orthogonal question: who owns resposibility for the canonical Azure DCOS Windows image? @anhowe do you know the answer to this?

If we want to add functionality to acs-engine that accepts a configurable image URL, the above change obviously won't do it (the above change seems to be literally modifying the download URL. What is it that we ultimately want to accomplish?

Reply:
We are trying to be able to develop the deployment process and binaries, potentially breaking it, without requiring a recompile on acs-engine for each developer.

Reply:
WRT the orthogonal question, we are providing a stable download URL for the  foreseeable future. I expect others will pick it up eventually, but for now this team is doing it. As DCOS on Windows is exclusive to vlabs, anything we provide is use at ones own risk, as development is ongoing.